internal_transfers/actions/index.ts

```diff
@@ -0,0 +1,22 @@
+import { ActionFn, Context, Event, TransactionEvent } from "@tenderly/actions";
+import { partitionEventLogs } from "./src/parse";
+
+export const triggerInternalTransfersPipeline: ActionFn = async (
+  context: Context,
+  event: Event
+) => {
+  const transactionEvent = event as TransactionEvent;
```

Comment:
This seems to transform the `event` object into a `TransactionEvent` object. This looks anti-pythonic, where a function is responsible for two different things: transform and trigger. In my opinion, I would expect this to be transformed before given to the function.

Reply:
It's not really being transformed, but rather cast to its actual type. I think @fleupold can say a bit more about this cast, as he introduced it I. The other tenderly action project. 

My understanding of this is that tenderly creates this "generic" function signature that takes Event (for multiple purposes) and in our case we know we're expecting a transaction event.